templates/repo/blame.tmpl

```diff
@@ -1,14 +1,9 @@
 <div class="{{TabSizeClass .Editorconfig .FileName}} non-diff-file-content">
-	<h4 class="file-header ui top attached header df ac sb">
-		<div class="file-header-left df ac">
-			<div class="file-info text grey normal mono">
-				<div class="file-info-entry">
-					{{.NumLines}} {{.locale.TrN .NumLines "repo.line" "repo.lines"}}
-				</div>
-				<div class="file-info-entry">{{FileSize .FileSize}}</div>
-			</div>
+	<h4 class="file-header ui top attached header df ac sb fw">
+		<div class="file-header-left df ac py-3 pr-4">
+			{{template "repo/file_info" .}}
```

Comment:
Was the template here previously outdated, or were the other entries missing on purpose?

Reply:
I think it was outdated and people forgot to keep blame fields in sync with view fields.